pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>fi.hsl</groupId>
     <artifactId>transitdata-common</artifactId>
-    <version>1.0.9</version>
+    <version>1.0.10</version>
```

Comment:
Convention has been that we don't bump version in the feature branch but in develop branch when we merge to master branch (the merge can contain several feature merges).